src/e2e/java/teammates/e2e/cases/lnp/InstructorStudentEnrollmentLNPTest.java

```diff
@@ -232,6 +232,7 @@ public void runLnpTest() throws IOException {
     public void classTearDown() throws IOException {
         // There is no need to add the newly enrolled students to the JSON DataBundle#students. This is because the new
         // CourseStudent entities that were created are automatically deleted when the corresponding course is deleted.
+        generateResultAnalysis("InstructorStudentEnrollmentLNPTest", 0.2, 60000);
```

Comment:
Can we do something similar to what is done with [getLnpTestPlan()](https://github.com/TEAMMATES/teammates/blob/master/src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java#L196) instead? And parameterise `0.2` and `60000` as overridden methods, and add details in Javadocs (eg. time should be specified in ms, not seconds)